<?xml version="1.0" encoding="UTF-8"?>
<launch>
  <arg name="robot_list" default="husky1,husky2,husky3"/>
  <arg name="aerial_map_ns" default="quadrotor"/>

  <arg name="output" default="screen"/>
  <arg name="viz" default="True"/>

  <param name="use_sim_time" value="True"/>

  <node pkg="semantics_manager" type="viz_goals.py" name="viz_goals" output="$(arg output)">
    <param name="~robot_list" value="$(arg robot_list)"/>
    <param name="~min_goal_dist_m" value="15"/>
  </node>

  <node type="rviz" name="rviz" pkg="rviz" args="-d $(find semantics_manager)/launch/sim_viz.rviz" if="$(arg viz)"/>
</launch>
